webpack/components/AnsibleHostDetail/components/AnsibleHostInventory/index.js

```diff
@@ -0,0 +1,24 @@
+import React, { useMemo } from 'react';
+import { foremanUrl } from 'foremanReact/common/helpers';
+import { useAPI } from 'foremanReact/common/hooks/API/APIHooks';
+import PropTypes from 'prop-types';
+import AnsibleHostInventory from './AnsibleHostInventory';
+
+const WrappedAnsibleHostInventory = ({ hostId }) => {
+  const params = useMemo(() => ({ params: { host_ids: [hostId] } }), [hostId]);
+
+  const url = foremanUrl('/ansible/api/ansible_inventories/hosts');
```

Comment:
This makes the url undefined if no hostId exists, no fake ajax calls will be made 
```suggestion
  const url = hostId && foremanUrl('/ansible/api/ansible_inventories/hosts');
```